os/arch/arm/src/common/up_internal.h

```diff
@@ -278,7 +278,7 @@ EXTERN uint32_t _edata;			/* End+1 of .data */
 EXTERN uint32_t _sbss;			/* Start of .bss */
 EXTERN uint32_t _ebss;			/* End+1 of .bss */
```

Comment:
Please correct the path for the changes described in the commit title
"os/arch/arm/up_internal.h" -> "os/arch/arm/src/common"